TestProject_Calc/LoginPageTest.cs

```diff
@@ -14,7 +16,6 @@ public void SetUpDriver()
             options.AddArgument("--headless");
             _driver = new ChromeDriver(options);
             _driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
```

Comment:
Default value for PageLoad is 300 seconds, so I think it's better to use smaller value.
```suggestion
            _driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
            _driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);```